doc/sphinx/conf.py

```diff
@@ -201,9 +201,9 @@
 # The empty string is equivalent to '%b %d, %Y'.
 #html_last_updated_fmt = None
 
-# If true, SmartyPants will be used to convert quotes and dashes to
-# typographically correct entities.
-html_use_smartypants = False # FIXME wrap code in <code> tags, otherwise quotesget converted in there
+# FIXME: this could be re-enabled after ensuring that smart quotes are locally
+# disabled for all relevant directives
```

Comment:
Indeed, it's a bit sad to lose smart quotes in many places. But as a quick fix it makes sense.